Classes/Processor/VideoProcessor.php

```diff
@@ -95,6 +97,24 @@ protected function renderYoutubeVideo(FileInterface $file): string
         );
     }
 
+    /**
+     * @param \TYPO3\CMS\Core\Resource\FileInterface $file
+     * @return string
+     */
```

Comment:
```suggestion
```

The doc block does not add anything to the method head. Therefore I suggest to remove it.

Reply:
You are right, I removed it